wolfcrypt/test/test.c

```diff
@@ -6392,8 +6400,13 @@ int ecc_test(void)
 
 #ifdef HAVE_ECC_SIGN
     /* test DSA sign hash */
-    for (i = 0; i < (int)sizeof(digest); i++)
+    for (i = 0; i < (int)sizeof(digest); i++) {
+    #ifdef ECC_TEST_VERIFY_ZERO
```

Comment:
Good point Jacob.  David after the release can you have it always test both cases?  Thanks.